android/src/com/mapswithme/util/bottomsheet/MenuBottomSheetFragment.java

```diff
@@ -0,0 +1,106 @@
+package com.mapswithme.util.bottomsheet;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.FragmentManager;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
+import com.mapswithme.maps.R;
+
+import java.util.ArrayList;
+
+public class MenuBottomSheetFragment extends BottomSheetDialogFragment
+{
+
+    private final Builder builder;
```

Comment:
I think we're using 2 spaces everywhere instead of 4.

Reply:
Oh yes forgot to change this in Android studio's config. Maybe you could provide an EditorConfig file that could be imported in Android Studio so future contributors would not need to tweak settings one by one?

Reply:
Check tools/android/formatter